tests/conftest.py

```diff
@@ -16,6 +17,11 @@ def default_session_fixture(request):
     logging.disable(logging.CRITICAL)
 
 
+@pytest.fixture(scope="session", autouse=True)
+def disallow_allowing_typed_generic_resources(request):
```

Comment:
What's request for?

Reply:
Nothing, it is used the signature for this method from the one above it for disabling logging. I hadn't checked if it works without tbh. Reading [the docs](https://docs.pytest.org/en/6.2.x/fixture.html) it can go and should work fine. I'll remove it 👍